edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
+    $id = $_GET['id'];
+    $query = "SELECT * FROM books WHERE id = $id";
+    $result = mysqli_query($connection, $query);
+    $book = mysqli_fetch_assoc($result);
+}
+
```

Comment:
- Observations: The SQL `UPDATE` statement is susceptible to SQL Injection since parameters are not bound.
- Recommendations: Leverage prepared statements and bound parameters to secure the update operation.

Example:
```php
$stmt = $connection->prepare("UPDATE books SET title = ?, author = ?, year = ? WHERE id = ?");
$stmt->bind_param("ssii", $title, $author, $year, $id);
$stmt->execute();
```